application/common/models/User.php

```diff
@@ -111,9 +126,11 @@ public static function findOrCreate($username = null, $email = null, $employeeId
             $user->idp_username = $personnelUser->username;
             $user->email = $personnelUser->email;
             if ( ! $user->save()) {
-                /*
-                 * add logging with model validation errors
-                 */
+                \Yii::error([
+                    'action' => 'create new user',
+                    'status' => 'error',
+                    'error' => $user->getFirstErrors(),
```

Comment:
no, since it gets serialized to json it works, and the email target serializes it with that vardumper class that writes it out nicely too